week_1/GymWebPage.css

```diff
@@ -4,16 +4,39 @@
     font-optical-sizing: auto;
     font-style: normal;
 }
+html{
+    overflow-x: hidden;
+    background:linear-gradient(135deg, #ffffff 0%,#4d5154 100%);
+}
+section{
+    padding-bottom: 300px;
+    padding-top: 100px;
+}
 header{
+    z-index: 100;
+    position: fixed;
+    top: 0;
     display: grid;
+    background:linear-gradient(135deg, #ffffff 0%,#4d5154 100%);
+    width: 100%;
     grid-template-columns: 1fr 1fr 2fr;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 6px -1px, rgba(0, 0, 0, 0.06) 0px 2px 4px -1px;
+}
+
+.logo{
+    background-color: aqua;
+}
+.logo_img{
+    position: absolute;
+    margin-top: 20px;
 }
 .title{
     display: inline;
     font-size: 25px;
-    position: relative;
+    position: absolute;
```

Comment:
@codal-uchoudhary , any specific reason we're using absolute over here ?

ideally the parent element should have a relative position and any corresponding children must have the absolute position.

CC: @codal-shubham